TaskManager/Modules/Home/View/HomeViewController.swift

```diff
@@ -7,13 +7,22 @@
 
 import UIKit
 
-enum Section: String, CaseIterable {
+enum Section: String {
     case Active
     case Completed
+    
+    func localizedString() -> String {
```

Comment:
Let's add this function to an extension of String so we can call it on any string we need